gdc_client/upload/client.py

```diff
@@ -139,7 +139,7 @@ def create_resume_path(file_path):
 
 class GDCUploadClient(object):
 
-    def __init__(self, token, processes, server, part_size,
+    def __init__(self, token, processes, server, upload_part_size,
```

Comment:
using empty dict as default parameter for `files`

Reply:
yeaah....